import-export-cli/utils/tokenManagement_test.go

```diff
@@ -142,7 +142,8 @@ func getOAuthStubOK(t *testing.T) *httptest.Server {
 			 "scopes":
 					["apim:api_view","apim:api_create","apim:api_publish",
 					 "apim:tier_view","apim:tier_manage","apim:subscription_view",
-					 "apim:subscription_block","apim:subscribe"
+					 "apim:subscription_block","apim:subscribe","apim:common_operation_policy_view",
```

Comment:
are these scopes correct. Do we need these? We only need apim:policy_import_export isn't it?

Reply:
Fixed by [34b09f4](https://github.com/wso2/product-apim-tooling/pull/924/commits/34b09f47c80ad3f6e3111bf2fa8b3b0b197ea4ea)